salt/roots/worker/service.sls

```diff
@@ -0,0 +1,11 @@
+resque worker service:
+  cmd.run:
+    - name: 'nohup bundle exec rake resque:work > /vagrant/log/resque.log 2> /vagrant/log/resque.err < /dev/null &'
+    - cwd: /vagrant
+    - unless: pgrep -fa resque
+    - env:
+      - DB_HOST: 192.168.69.20
```

Comment:
Ah, I'll bet you're right! I've pulled your branch for testing now. I rebased the changeset on the upstream master, so I'll push any additional fixups to my own remote. I will let you resolve your upstream branch accordingly.